numerics_paper/version2/numerical_chj.tex

```diff
@@ -652,7 +678,23 @@ \subsection{Entropy stabilization}\label{sec:entropy_stable}
   via a numerical experiment in \S \ref{sec:rp_dry_bed}.
 \end{remark}
 
-\clearpage
+\begin{remark}[About the design philosophy.]
```

Comment:
Since we don't refer to this, we don't need it as a numbered remark.

Reply:
I removed the number in the remark. I also wonder if we should put this in the main text instead of a remark.